setup.py

```diff
@@ -14,16 +8,21 @@
 from subprocess import call
 
 import setuptools.command.build_py
-from coalib.misc.BuildManPage import BuildManPage
 from setuptools import find_packages, setup
 from setuptools.command.test import test as TestCommand
 
+from coalib import assert_supported_version, VERSION, get_version
```

Comment:
should we sort these alphabetically? `assert_supported_version, VERSION, get_version`